grove/grovetccfg/grovetccfg.go

```diff
@@ -202,11 +207,75 @@ func main() {
 		}
 	}
 
+	// TODO remove this once converted to 1.3 API
+	// using the 1.2 API
+	var hostName string
+	var hostServer tc.Server
+	var hostProfile tc.Profile
+	var ok bool
+	var profiles map[string]tc.Profile
+	var servers map[string]tc.Server
```

Comment:
If `host == nil`, this gets passed to `createRulesOldAPI` which will panic. This should check if `host == nil`, and print a good error message and `os.Exit(1)`.

That will also reduce indentation, and reduce variable pre-declarations too, which will make this block much nicer.